### PR TITLE
Change box type to show signIn style and add 1 props which is contentStyle

### PR DESCRIPTION
**Pull request by DevYuns**
## Description

Change box type to show signIn style and add 1 props which is contentStyle.
I'm checking style on web, ios, android and storybook.
And I will update README.md file If this PR should be merged.

The new style shows below.
image

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by DevYuns** [comment]
Ci test failed because of other component test snapshot.

**Comment by hyochan** [comment]
> Ci test failed because of other component test snapshot.

Nam Tat?

**Review by hyochan — CHANGES_REQUESTED** [review]
image

You only updated the `boxType`. Please fix others too. You can mostly participate on the colors.

**Comment by DevYuns** [comment]
Ok I will

**Comment by hyochan** [comment]
> Ok I will

When?

**Comment by DevYuns** [comment]
I've try to change the` ternary operator` to `&&` operator in below image1 section, It induced errors in mobile. This errors can be occurred only about Text component in JSX. If someone wants to know further about this problem, check [here](https://github.com/facebook/react-native/issues/20764).  

image1
Screen Shot 2020-07-30 at 11 05 17


Try
Screen Shot 2020-07-30 at 11 05 43


Error message
Screen Shot 2020-07-30 at 11 05 57

**Review by hyochan — CHANGES_REQUESTED** [review]
Please re-request review when you are perfect

**Review by hyochan — CHANGES_REQUESTED** [review]
image

File: main/EditText/README.md [hunk 1]
```diff
@@ -54,86 +55,115 @@ yarn add dooboo-ui
...
+              fontWeight: 'bold',
+              fontSize: 24,
+              lineHeight: 35,
+              color: '#495057',
+            }}>
+            Sign in with Email
+          </Text>
           <EditText
-            testID="EMAIL_INPUT"
+            testID="EMAIL_INPUT_DEFAULT"
```
Comment (hyochan) [inline main/EditText/README.md]: Still not updated!

Comment (hyochan) [reply main/EditText/README.md]: Update the readme too!

**Review by hyochan — APPROVED** [review]

**Pull request merged**